recipe/conda_build_config.yaml

```diff
@@ -1,14 +1,7 @@
-c_compiler:          # [unix]
-  - clang_bootstrap  # [osx]
-  - gcc              # [linux]
-cxx_compiler:        # [unix]
-  - clang_bootstrap  # [osx]
-  - gxx              # [linux]
-
-c_compiler_version:     # [osx]
-  - "*"                 # [osx]
-cxx_compiler_version:   # [osx]
-  - "*"                 # [osx]
+c_compiler:             # [osx]
+  - clang_bootstrap     # [osx]
+cxx_compiler:           # [osx]
+  - clang_bootstrap     # [osx]
 
 channel_targets:
   - conda-forge llvm_rc
```

Comment:
Dev versions should go into `llvm_dev`.

Reply:
OK, will change that if I do any other builds (which I don't expect). Basically I wanted to do an "rc0" just to sort out potential problems, so that for rc1[^1], we essentially just have to bump the hash (hence why I kept things rc-ish...).

[^1]: which is more or less also just randomly cut from main, as [explained](https://www.youtube.com/watch?v=onaNb2U1Od8) by their release manager